resources/build/build-utils.sh

```diff
@@ -190,10 +169,12 @@ function findShouldSentryRelease() {
     esac
 }
 
-findRepositoryRoot
-_builder_setBuildScriptIdentifiers
+findKeymanRoot
 findTier
 findVersion
+
+# Source builder_script
+. "$KEYMAN_ROOT/resources/build/builder.inc.sh"
```

Comment:
builder.inc.sh should be in resources/, not resources/build/. We are slowly deprecating use of the resources/build/ folder as .../build/ is a standard build output folder, and so it's confusing to use it for scripts.